docs/properties-reference.md

```diff
@@ -2176,8 +2178,8 @@ ldap.synchronization.defaultHomeFolderProvider | The default home folder provide
 | transferservice.receiver.stagingDir | Transfer Service | ${java.io.tmpdir}/alfresco-transfer-staging |
 | transformer.debug.entries | Number of entries to show in debug output | 0 |
 | transformer.log.entries | Number of entries to show in the log | 50 |
-| transform.misc.startupRetryPeriodSeconds | When the legacy misc transformer .url is set, this value indicates the amount of time to wait after a connection failure before retrying the connection to allow a docker container to (re)start. | 60 |
-| transform.misc.url | Legacy misc transformer url to T-Engines to service transform requests via http. Disabled by default. | |
+| ~~transform.misc.startupRetryPeriodSeconds~~ | ~~When the legacy misc transformer .url is set, this value indicates the amount of time to wait after a connection failure before retrying the connection to allow a docker container to (re)start.~~ | ~~60~~ |
+| ~~transform.misc.url~~ | ~~Legacy misc transformer url to T-Engines to service transform requests via http. Disabled by default.~~ | |
 | transform.service.cronExpression | Schedule for reading local transform config, so that T-Engines and local pipeline config is dynamically picked up, or reintegrated after an outage. Initially checks every 10 seconds and then switches to every hour after the configuration is read successfully. If there is a error later reading the config, the checks return to every 10 seconds. | `6 30 0/1 * * ?` |
 | transform.service.enabled | Determines whether the transform service is enabled | false |
 | transform.service.initialAndOnError.cronExpression | | `0/10 * * * * ?` |
```

Comment:
Maybe we could add a NOTE just after the table to say that strikethrough'd properties have been removed in 7.0 (that will at least buy us a little time until we remove more in 7.1 ;-) )

Furthermore, the intention was for this list to move to the official docs where there are version specific pages so I think the note will be sufficient for now.

Reply:
* There already is a note at the top of the table about strikethrough text. Originally it was at the end, but as it is such a long page, it is not seen.

* I was talking with on Adelaide about transform properties on Monday (11 Jan), and pointed her at this list. I don't think they have been included yet in the official docs.